tests/cli/cli.test.ts

```diff
@@ -37,19 +37,22 @@ describe('Manage starknet account', function () {
   this.timeout(TIME_LIMIT);
 
   it('should create a starknet account', async () => {
-    if (fs.existsSync(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'))) {
-      fs.unlinkSync(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'));
+    try {
+      await fs.access(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'));
+      await fs.unlink(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'));
+    } catch {
+      // file does not exist
```

Comment:
warning

Reply:
I don't think it's necessary in this case, wdyt? This is almost equivalent to before, since the idea is `fs.access` throws if it can't find the directory. Still - I do think an `exists` helper would be nice here, since an empty catch does not look that good and may hide an error from `unlink` (we could still check the error, but a helper would be better IMO).